config/hibernator.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: pincher.devtron.ai/v1alpha1
```

Comment:
Can we include some regular use case hibernator examples? A hibernator named test is not helping people to understand what exactly it does. Some cases to cover:
- weekend hibernator
- night-time hibernator
- temporary deployments (delete after 5 days)